{"name": "test_2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 3", "trace": "testcase\\test_fixture.py:42: in test_2\n    assert 2==3\nE   assert 2 == 3"}, "start": 1758268351966, "stop": 1758268351966, "uuid": "060ff57f-573e-4832-a5fb-d1e1a60bc758", "historyId": "9b0754076bb05a19eaad952a5f0e2e1d", "testCaseId": "9b0754076bb05a19eaad952a5f0e2e1d", "fullName": "testcase.test_fixture.TestFixtureOne#test_2", "labels": [{"name": "story", "value": "usefixture的使用"}, {"name": "feature", "value": "fixture的使用"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_fixture"}, {"name": "subSuite", "value": "TestFixtureOne"}, {"name": "host", "value": "DESKTOP-JBGUQCK"}, {"name": "thread", "value": "14508-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_fixture"}], "titlePath": ["testcase", "test_fixture.py", "TestFixtureOne"]}